projects/addon-mobile/components/sheet/components/sheets-host/sheets-host.component.ts

```diff
@@ -1,11 +1,17 @@
 import {AnimationOptions} from '@angular/animations';
-import {ChangeDetectionStrategy, Component, Inject} from '@angular/core';
-import {TUI_WINDOW_HEIGHT} from '@taiga-ui/cdk';
+import {
+    ChangeDetectionStrategy,
+    ChangeDetectorRef,
+    Component,
+    Inject,
+    Self,
+} from '@angular/core';
+import {TUI_WINDOW_HEIGHT, TuiDestroyService} from '@taiga-ui/cdk';
 import {TUI_ANIMATION_OPTIONS, tuiFadeIn, tuiSlideInTop} from '@taiga-ui/core';
 import {Observable} from 'rxjs';
-
 import {TuiSheet} from '../../sheet';
 import {TuiSheetService} from '../../sheet.service';
+import {takeUntil} from 'rxjs/operators';
```

Comment:
Looks like you need to run lint fix to order imports. It should have been ran automatically in precommit hook. Don't skip them :)

Reply:
I didn't skip them. Seems like whatever is setup didn't work for me. Maybe because I'm running Windows.